.gitignore

```diff
@@ -7,6 +7,15 @@
 bin/**
 dist/**
 
+# IDE specific
```

Comment:
It's subjective, but I like having these general files in the projects gitignore as it can make it easier for people to contribute without having to figure out a global gitignore if they haven't already